cfpb/cfgov-refresh
# Missing service member templates

These are missing service-member templates which were missed when crawling because they aren't linked. 

## Additions

-

## Removals

-

## Changes

- Modified urls.py, templates, and images files to display missing templates.

## Testing

1. Visit `http://localhost:8000/servicemembers/additionalresources/` and ensure the page and links work properly
2. Visit `http://localhost:8000/servicemembers/protecting/` and ensure the page and links work properly.
3. Visit `http://localhost:8000/servicemembers/planning/index.html` and ensure the page and links work properly.
4. Visit `http://localhost:8000/servicemembers/planning/creativesavingsstrategies/` and ensure the page and links work properly.

## Screenshots


## Notes

-

## Todos

-

## Checklist
